lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2653,7 +2643,8 @@ CompilerType TypeSystemSwiftTypeRef::GetChildCompilerTypeAtIndex(
   if (get_ast_num_children() <
       runtime->GetNumChildren({this, type}, valobj).getValueOr(0))
     return impl();
-
+  if (ShouldSkipValidation(type))
+    return impl();
```

Comment:
Why is this necessary? Isn't this check part of the VALIDATE_AND_RETURN macro below?